app/models/bishop.rb

```diff
@@ -1,2 +1,7 @@
 class Bishop < Piece
+
+  def valid_move?(x, y) #NO CAPTURE/COLLISION LOGIC YET
```

Comment:
I like what you were going for here with the inline comments. Here's my suggestion if you do actually want to make your code look badass and readable:

Inline comments are typically unnecessary. The code should speak for itself in the sense that your variable/method names should all be verbose and the logic should not nested deep in data structures. It should be assumed that any developer reading a method and understand the fairly small amount of lines in it.

What is important is called the Doc Block. A Doc Block describes what a method does, what arguments (types and definitions) it accepts, and what is returned. This helps developers understand your method in relation to all the code without having to read the method.

The reason one is important (IMO) and not the other is that if you are in the context of a function, you are already in the context of the function. You've already likely read the code leading up to where you are, and you completely understand the variables that are in play.

When you're in a different method though, you don't want to have to read another method's code. You want to be able to get what you need (typically IDEs parse this automatically and provide in your context hints). 

http://guides.rubyonrails.org/api_documentation_guidelines.html